all/001-kolla-defaults.yml

```diff
@@ -56,7 +56,7 @@ kolla_base_distro: "rocky"
 kolla_internal_vip_address: "{{ kolla_internal_address | default('') }}"
 kolla_internal_fqdn: "{{ kolla_internal_vip_address }}"
 kolla_external_vip_address: "{{ kolla_internal_vip_address }}"
-kolla_same_external_internal_vip: "{{ kolla_external_vip_address == kolla_internal_vip_address }}"
+kolla_same_external_internal_vip: "{{ kolla_external_vip_address | ansible.utils.ipaddr('address') == kolla_internal_vip_address | ansible.utils.ipaddr('address') }}"
```

Comment:
This file is in sync with https://github.com/openstack/kolla-ansible/blob/master/ansible/group_vars/all.yml#L59. Change it there first.